src/runtime/composables/useMedusaClient.ts

```diff
@@ -9,9 +9,11 @@ export const useMedusaClient = (): Medusa => {
   // If medusa was registered in global plugin, just return reference to it.
   if (config.global) return nuxtApp.$medusa
 
+  const { medusa: privateConfig } = useRuntimeConfig()
```

Comment:
We want to have the api key from the private config only accessible from the server utility, not the frontend client.

Take a look at how I have implemented that in the security module for basic auth that is passed as private config :) 

https://github.com/Baroshem/nuxt-security/blob/main/src/runtime/server/middleware/basicAuth.ts

Reply:
Sorry for my mistake. And thank you for the guide.
I move the private config to `serverMedusaClient` for server utility.
I'm not sure that is the right place. By the way, there is no `apiKey` or any private expose to client.

![image](https://user-images.githubusercontent.com/6861191/231540571-21a5638c-c23d-42ed-b31f-a8705c463e13.png)

Here is the commit: 7e61448773f3d12c0fea92c5ca41b91f7654e973